preset/browser/jest-preset.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   ...shared,
   // uses a webpack style resolver, the default one has many issues.
   resolver: require.resolve("./resolver"),
+  testEnvironment: 'jsdom',
```

Comment:
In Jest 27, `jsdom` is no longer the default test environment, so we need to specify it here.